www-src/app/replicator/replicator_backups.coffee

```diff
@@ -124,15 +126,16 @@ module.exports =
         async.series [
             @ensureDeviceFolder.bind this
             ImagesBrowser.getImagesList
-            (callback) => @photosDB.query 'PhotosByLocalId', {}, callback
-            (cb) => @db.query 'FilesAndFolder',
+            (callback) =>
```

Comment:
@kosssi you should rename `callback` in  `cb` to be consitent with the new function (and avoid confusion with the `callback` from the outer function syncPictures).